pmix/borrow.py

```diff
@@ -23,9 +23,8 @@
 Author: James K. Pringle
 E-mail: jpringle@jhu.edu
 """
-
 import argparse
-import os.path
+import os
```

Comment:
This should be `import os.path`

Reply:
Why is that? Code works fine with out `os.path`.  `import os.path` throws a warning in my IDE. I don't remember what it is at the moment but I'll look it up for you.

Reply:
Your IDE highlights it with a warning? I wonder what is different between our PyCharms. I don't have a warning highlight. Also, the reason is explained in the accepted answer here: https://stackoverflow.com/questions/2724348/should-i-use-import-os-path-or-import-os